plugins/dev-create/templates/plugin/template.json

```diff
@@ -5,8 +5,10 @@
     "build": "blockly-scripts build",
     "clean": "blockly-scripts clean",
     "lint": "blockly-scripts lint",
+    "predeploy": "blockly-scripts predeploy",
```

Comment:
These should be updated for all template.jsons (field, block, etc)

Reply:
Okay, added it to all types and the create script. Also added auditFix to the create script.